src/core/Akka.Remote/Transport/Helios/HeliosTcpTransport.cs

```diff
@@ -207,18 +155,17 @@ public TcpAssociationHandle(Address localAddress, Address remoteAddress, HeliosT
 
         public override bool Write(ByteString payload)
         {
-            if (_channel.IsOpen())
+            if (_channel.IsOpen && _channel.IsWritable)
```

Comment:
If the channel is not writable (too many pending writes queued up already) then this will trigger Akka.Remote's adaptive backoff mechanism to run - and that's a good thing.